src/main/java/frc/robot/Robot.java

```diff
@@ -83,63 +108,73 @@ public void robotPeriodic() {
    */
   @Override
   public void autonomousInit() {
-    this.autonomousCommand = new AutonomousCommand(distance, maxSpeed, drive, pValue);
     m_autoSelected = m_chooser.getSelected();
-    switch(m_autoSelected) {
-      case kCustomAuto:
-        break;
-      case kDefaultAuto:
-        default:
-          if (autonomousCommand != null) {
-            autonomousCommand.schedule();
-          }
-        break;
+    switch (m_autoSelected) {
+    case kCustomAuto:
+      break;
+    case kDefaultAuto:
+    default:
+      if (autonomousCommand != null) {
+        autonomousCommand.schedule();
+      }
+      break;
     }
   }
 
-  /**
-   * This function is called periodically during autonomous.
-   */
   @Override
   public void autonomousPeriodic() {
     CommandScheduler.getInstance().run();
   }
 
   @Override
   public void teleopInit() {
-    this.teleopCommand = new TeleopCommand(controls, drive, pValue);
-    if (teleopCommand != null) {
-      teleopCommand.schedule();
-    }
   }
- 
+
+  // use this to override the algorithm and just use a speed
+  @Config
+  public void setOverrideSpeed(double overrideSpeed) {
+    this.overrideSpeed = overrideSpeed;
+    shooterSpeedController.setLaunchSpeed(this.overrideSpeed);
+  }
+
   /**
    * This function is called periodically during operator control.
    */
   @Override
   public void teleopPeriodic() {
     CommandScheduler.getInstance().run();
+
+    if (controls.getLeftTrigger()) {
+      indexerController.outtake();
+    }
```

Comment:
We should break this out into it's own command as well.

Reply:
Ok, but let's make that a new feature.  It should be nice and small and will teach someone (Garrett?) a bit about Commands (and if we make indexerController a SubsystemBase they will learn about interruption as well).

Reply:
I am good with that